[2025-02-09T08:33:34.139+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-09T08:33:34.169+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-02-09T08:33:30.548862+00:00 [queued]>
[2025-02-09T08:33:34.185+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-02-09T08:33:30.548862+00:00 [queued]>
[2025-02-09T08:33:34.186+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-09T08:33:34.206+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_directory> on 2025-02-09 08:33:30.548862+00:00
[2025-02-09T08:33:34.219+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_directory', 'manual__2025-02-09T08:33:30.548862+00:00', '--job-id', '1377', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmpebdgfqln']
[2025-02-09T08:33:34.222+0000] {standard_task_runner.py:105} INFO - Job 1377: Subtask check_directory
[2025-02-09T08:33:34.217+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=286) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-09T08:33:34.224+0000] {standard_task_runner.py:72} INFO - Started process 288 to run task
[2025-02-09T08:33:34.293+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-02-09T08:33:30.548862+00:00 [running]> on host 003b7fbea265
[2025-02-09T08:33:34.387+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T08:33:30.548862+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-09T08:33:30.548862+00:00'
[2025-02-09T08:33:34.388+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-09T08:33:34.389+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-09T08:33:34.389+0000] {logging_mixin.py:190} INFO - Current task name:check_directory state:running start_date:2025-02-09 08:33:34.170034+00:00
[2025-02-09T08:33:34.390+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-02-09T08:33:34.391+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-09T08:33:34.391+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-09T08:33:34.393+0000] {python.py:240} INFO - Done. Returned value was: check_file_in_directory
[2025-02-09T08:33:34.393+0000] {branch.py:38} INFO - Branch into check_file_in_directory
[2025-02-09T08:33:34.394+0000] {skipmixin.py:230} INFO - Following branch ('check_file_in_directory',)
[2025-02-09T08:33:34.631+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-09T08:33:34.660+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-09T08:33:34.661+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_directory, run_id=manual__2025-02-09T08:33:30.548862+00:00, execution_date=20250209T083330, start_date=20250209T083334, end_date=20250209T083334
[2025-02-09T08:33:34.673+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-09T08:33:34.673+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-09T08:33:34.674+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-02-09 08:33:30.583876+00:00
[2025-02-09T08:33:34.675+0000] {logging_mixin.py:190} INFO - Task hostname:003b7fbea265 operator:BranchPythonOperator
[2025-02-09T08:33:34.721+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-09T08:33:34.769+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-09T08:33:34.771+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
